master/include/logic/checkupManager.hpp

```diff
@@ -195,7 +206,7 @@ inline CheckupManager::CheckupError CheckupManager::initialCheckupSequence(Digit
             // digitalSender->toggleWatchdog();
         // TS Activated?
             if (_systemData->failureDetection.ts_on) {
-                checkupState = CheckupState::TOGGLE_VALVE;
+                checkupState = CheckupState::CHECK_TIMESTAMPS;
```

Comment:
Why?

Reply:
Forgot to change it after testing, I'll change it. Thank you